docs/content/annotations.md

```diff
@@ -238,6 +242,21 @@ allow {
 }
 ```
 
+#### Inlined Schema Format
+
+Schema definitions can be inlined by specifying the schema structure as a YAML- or JSON map.
```

Comment:
`YAML-` -> `YAML`

Reply:
This is an intentional suspended hyphen to tie together `YAML` and `map`.

But perhaps that is only allowed for closed compound words and hyphenated compound words .. 🤔. E.g. `YAML- and JSON-map`

Reply:
Yes... It would look great in Swedish though 😄